src/main/java/calculator/CalculatorApplication.java

```diff
@@ -0,0 +1,29 @@
+import calculator.Calclator;
+import calculator.Spliter;
+
+import java.util.Scanner;
+
+
+public class CalculatorApplication {
+    public static void main(String[] args) {
+        int result = 0;
+        System.out.println("숫자들을 입력하세요 : ");
+        Scanner scanner = new Scanner(System.in);
+        String inputValue = scanner.nextLine();
+        if (inputValue.isEmpty()) {
+            System.out.println(result);
+            return;
+        }
+
+        Spliter spliter = new Spliter();
```

Comment:
Splitter 가 Main 에서도 생성되고 Calculator 에도 생성되는 것 같아요!
불필요하게 생성되고 있는 것은 아닌지 생각해보아야할 것 같습니다. 

만약 Calculator 에서 Splitter가 꼭 필요하다면 Main 에서 생성한 Splitter를 아래처럼 Calculator 로 주입해서 쓸 수 있을 것 같아요.
의존성을 외부에서 넣어주는 것인데 DI 패턴이라고도 합니다!(https://gmlwjd9405.github.io/2018/11/09/dependency-injection.html)

`Calculator calculator = new Calculator(splitter)` 

```suggestion
    private final Spliter spliter;
    
    public Calculator(Splitter splitter){
           this.splitter = splitter;
    }
```

또는, 혹시 Calculator 가 Splitter 의존성을 불필요하게 들고있다거나, Splitter 의 역할이 잘못 되어있는 것은 아닌지(위에 민정이 리뷰!) 생각해보시면 좋을 것 같습니다